var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -117,6 +125,11 @@ def cmake_args(self):
             '-DXERCESC_ROOT_DIR={0}'.format(spec['xerces-c'].prefix)
         ]
 
+        # Don't install the package cache file as Spack will set
+        # up CMAKE_PREFIX_PATH etc for the dependencies
+        if spec.version > Version('10.5.99'):
```

Comment:
Just out of curiosity:
Why are you not using `spec.satisfies('@10.5.99:')`?
or even `'@10.6:'?

Reply:
FWIW I do this because it's easier to read and more robust against typos. It's especially nice if you want open-ended versions -- `'@10.6.0:10.6.99'` is less sensible and more readable than `Version('10.6') <= spec.version < Version('10.7')` IMHO.